tests/output/test_plotting.py

```diff
@@ -1,13 +1,19 @@
 import pytest
-from firedrake import *
-from firedrake.plot import FunctionPlotter
-import matplotlib.pyplot as plt
-import matplotlib.colors
-from matplotlib.animation import FuncAnimation
-from mpl_toolkits.mplot3d import Axes3D
 import numpy as np
+from firedrake import *
+
+try:
+    from firedrake.pyplot import *
+    import matplotlib.pyplot as plt
+    import matplotlib.colors
+    from matplotlib.animation import FuncAnimation
+    from mpl_toolkits.mplot3d import Axes3D
+except ImportError:
+    # Matplotlib is not installed
```

Comment:
Arguably easier to have `pytest.skip("Matplotlib is not installed", allow_module_level=True)` (https://docs.pytest.org/en/6.2.x/skipping.html#skipping-test-functions) here instead of adding all the individual skip markers.